{"name": "User login with incorrect credentials", "status": "passed", "attachments": [{"name": "stdout", "source": "c45f98f1-0d0c-4bea-826f-33de9801b22d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "payload", "value": "{'password': 'yozhik-123'}"}], "start": 1732029586047, "stop": 1732029586825, "uuid": "af316e7f-19a7-4fe8-8457-b3d732d8c711", "historyId": "baf1c10f0904007a94cdfe2c008936f7", "testCaseId": "76148a48b7585b4a8f8a20e34c40f04e", "fullName": "tests.user.test_user_login.TestUserLogin#test_user_login_incorrect_credentials", "labels": [{"name": "suite", "value": "User Tests"}, {"name": "parentSuite", "value": "API Tests"}, {"name": "subSuite", "value": "User Login Tests"}, {"name": "host", "value": "ubuntu22"}, {"name": "thread", "value": "10772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.user.test_user_login"}]}